addons/binding/org.openhab.binding.kostal.inverter/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/jsoup-1.8.3.jar
```

Comment:
Could you add an about.html similar to [this one](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.freebox/about.html#L28), which mentions the included lib and its license?